invenio_i18n/ext.py

```diff
@@ -62,22 +63,22 @@ def default(self, o):
 
 
 class InvenioI18N(object):
-    """Invenio I18N extension.
-
-    :param app: Flask application.
-    :param data_formats: Override default date/time formatting.
-    :param localeselector: Callback function used for locale selection.
-        Defaults to using :py:func:`invenio_i18n.selectors.get_locale()`.
-    :param timezoneselector: Callback function used for timezone selection.
-        Defaults to ``BABEL_DEFAULT_TIMEZONE``.
-    :param entrypoint: Entrypoint used to load translations from. Set to
-        ``None`` to not load translations from entry points.
-    """
+    """Invenio I18N extension."""
 
     def __init__(self, app=None, date_formats=None, localeselector=None,
                  timezoneselector=None,
                  entry_point_group='invenio_i18n.translations'):
-        """Initialize extension."""
+        """Initialize extension.
+
+        :param app: Flask application.
```

Comment:
Is this rendered when built? I vaguely remember something about having to add it in the class doc to have it show up.